.github/workflows/tests-integration-no-db.yml

```diff
@@ -53,9 +53,9 @@ jobs:
           poetry-version: 2.1.2
 
       - name: Poetry install
-        run: poetry run -- nox -s run_in_dev_env -- poetry install
+        run: poetry run -- nox -s devenv:run -- poetry install
 
       - name: Run Python integration tests without db
-        run: poetry run -- nox -s run_python_test -- ${{ matrix.test-path.path }}
+        run: poetry run -- nox -s devenv:pytest -- ${{ matrix.test-path.path }}
```

Comment:
might duplicate another comment's underlying topic:
why  not use `itests:no-db` here? or have the same series of steps & similar nox sessions in `.github/workflows/tests-integration-db.yml`?